### ISSUE TITLE
Release procedure: when is a ticket mature enough

### ISSUE THREAD

**Issue by alamers** [adewg/ICAR#207]
While preparing for the 1.2 release, the question popped up when a ticket is finished enough. Right now, if there is a json scheme defined for a message and the content is approved in the biweekly meeting, it is merged into `Develop`. Are there any other checks that we want to perform before we release it into the wild?

Things that come to mind:
* run a code generator to see if it works as expected
* have at least one company working on an implementation that confirms the feasibility of that new message
* have a freeze period where interested parties can give a last-minute comment

The code generator check seems easy enough to do just before a release. The other options appear to be a chicken-and-egg problem since most companies will start only after an official release. 

While doing a patch release is fine, there is a chance that it may have to contain breaking changes. Do we accept that risk since most likely it is then the first implementation anyway (and thus not so breaking after all)?

**Comment by alamers** [comment]
As discussed in todays meeting:
* **code generator:** Running the code generator is a minimal effort quality check. We need to create a list of 'tested generators' that we support. For now that is the [OpenAPI Tools generator](https://github.com/OpenAPITools/openapi-generator) for the Java and C# languages. Running these checks will be added to the release procedure.
* **working implementation:** Since there is always a company driving a definition of a new message, that seems 'good enough' for now. We also have the chicken-and-egg problem where a lot of companies are only willing to invest development time when a message is 'released'. Using the 'minimal-design-and-extend-later' approach would alleviate this problem as well.
* **freeze period:** the current community is quite small and most active members are in the working group. A freeze period is not likely to yield any additional comments. As such, we do not use a freeze period right now.
